src/main.rs

```diff
@@ -23,21 +24,28 @@ fn init_logger() {
 }
 
 fn main() -> Result<(), String> {
-    init_logger();
+    let config = configuration::get_configuration();
+
+    if config.nun_user.as_str() == "" || config.nun_pwd == "" {
```

Comment:
This is problematic because, because user and PWD can also be provided via command line parameter

Reply:
so the priority is command line then env... will put that into a startup_condition_check function